vision_zero/schools.py

```diff
@@ -120,7 +115,17 @@ def pull_from_sheet(con, service, year, *args):
     LOGGER.info('Table %s is done', table)
 
 if __name__ == '__main__':
-    """The following connects to the database, establishes connection to the sheets and executes function based on the year of data required."""
+    """The following connects to the database, establishes connection to the sheets 
+    and executes function based on the year of data required.
+    Note: this part is not run by airflow.
+
+    """
+
+    SERVICE_ACCOUNT_FILE = '/home/jchew/bdit_data-sources/vision_zero/key.json' 
+
+    credentials = service_account.Credentials.from_service_account_file(
```

Comment:
this should be renamed to `service` since that's what gets sent to the `pull_from_sheet` function below

Reply:
There is already a `service` defined so I'm confused. Is `pull_from_sheet()` supposed to read in `credentials` or `service` as defined by `service = build('sheets', 'v4', credentials=credentials, cache_discovery=False)`?

Here's the whole block:
```
if __name__ == '__main__':
    """The following connects to the database, establishes connection to the sheets 
    and executes function based on the year of data required.
    Note: this part is not run by airflow.

    """

    SERVICE_ACCOUNT_FILE = '/home/jchew/bdit_data-sources/vision_zero/key.json'

    credentials = service_account.Credentials.from_service_account_file(
        SERVICE_ACCOUNT_FILE, scopes=SCOPES)

    CONFIG = configparser.ConfigParser()
    CONFIG.read(r'/home/cnangini/googlesheets_db.cfg')
    dbset = CONFIG['DBSETTINGS']
    con = connect(**dbset)

    service = build('sheets', 'v4', credentials=credentials, cache_discovery=False)

    pull_from_sheet(con, service, 2018)
    pull_from_sheet(con, service, 2019)
    pull_from_sheet(con, service, 2020)
```

Reply:
🤦‍♂️ sorry GitHub hides code blocks and I missed that. Nothing to change here.